docs/source/daml/intro/2_Scenario.rst

```diff
@@ -47,6 +47,7 @@ Just like ``Scenario`` is a recipe for a test, ``Update`` is a recipe for a tran
 
 You'll learn all about the syntax ``Token with owner = alice`` in :doc:`3_Data`.
 
+.. At this point I’m wondering if I need to include parantheses for it to be read as a single Update.
```

Comment:
The parentheses are needed because in DAML function application is left associative. So without the brackets it would be ``(submit alice create) Token with owner = alice`` which would result in a type error.